projects/js-packages/charts/src/stories/sample-data/index.ts

```diff
@@ -0,0 +1,712 @@
+		previousShare: 16,
+		delta: -7.9,
+		imageColor: '#00ff00',
+	},
+];
+
+/**
+ * Conversion funnel sample data
+ */
+export const sampleFunnelData = [
+	{
+		id: 'sessions',
+		label: 'Sessions',
+		rate: 100,
+		count: 10000,
+	},
+	{
+		id: 'cart',
+		label: 'Cart',
+		rate: 71.1,
+		count: 7110,
+	},
+	{
+		id: 'checkout',
+		label: 'Checkout',
+		rate: 52.5,
+		count: 5250,
+	},
+	{
+		id: 'purchase',
+		label: 'Purchase',
+		rate: 10.3,
+		count: 1030,
+	},
+];
+
+export const lowConversionData = [
+	{
+		id: 'sessions',
+		label: 'Sessions',
+		rate: 100,
+		count: 5000,
+	},
+	{
+		id: 'cart',
+		label: 'Cart',
+		rate: 45.2,
+		count: 2260,
+	},
+	{
+		id: 'checkout',
+		label: 'Checkout',
+		rate: 28.8,
+		count: 1440,
+	},
+	{
+		id: 'purchase',
+		label: 'Purchase',
+		rate: 6.4,
+		count: 320,
+	},
+];
+
+export const highConversionData = [
+	{
+		id: 'sessions',
+		label: 'Sessions',
+		rate: 100,
+		count: 8000,
+	},
+	{
+		id: 'cart',
+		label: 'Cart',
+		rate: 85.3,
+		count: 6824,
+	},
+	{
+		id: 'checkout',
+		label: 'Checkout',
+		rate: 72.1,
+		count: 5768,
+	},
+	{
+		id: 'purchase',
+		label: 'Purchase',
+		rate: 18.7,
+		count: 1496,
+	},
+];
+
+/**
+ * Bar list chart sample data
+ */
+export const barListSample = [
+	{
+		group: 'primary',
+		label: 'Jan 21-Aug 8, 2024',
+		data: [
+			{ label: 'Organic search', value: 30000 },
+			{ label: 'Affiliates', value: 19000 },
+			{ label: 'Display', value: 18000 },
+			{ label: 'Organic shopping', value: 16000 },
+		],
+	},
+	{
+		group: 'comparison',
+		label: 'Jan 21-Aug 8, 2023',
+		data: [
+			{ label: 'Organic search', value: 20000 },
+			{ label: 'Affiliates', value: 15000 },
+			{ label: 'Display', value: 19900 },
+			{ label: 'Organic shopping', value: 20500 },
+		],
+	},
+];
+
+export const salesByProduct = [
+	{
+		group: 'primary',
+		label: 'Sales By Product',
+		data: [
+			{ label: 'Behemoth hat ', value: 32400 },
+			{ label: 'Margarita top', value: 20000 },
+			{ label: 'Berlioz dress', value: 15000 },
+			{ label: 'Woland shirt', value: 16000 },
+			{ label: 'Azazello top', value: 20000 },
+		],
+	},
+];
+
+// Pie chart sample data
+export const operatingSystemsData = [
+	{
+		label: 'MacOS',
+		value: 30000,
+		valueDisplay: '30K',
+		percentage: 23,
+	},
+	{
+		label: 'Linux',
+		value: 22000,
+		valueDisplay: '22K',
+		percentage: 17,
+	},
+	{
+		label: 'Windows',
+		value: 80000,
+		valueDisplay: '80K',
+		percentage: 60,
+	},
+];
+
+// Semi-circle pie chart sample data (different percentages)
+export const operatingSystemsSemiCircleData = [
+	{
+		label: 'MacOS',
+		value: 30000,
+		valueDisplay: '30K',
+		percentage: 5,
+	},
+	{
+		label: 'Linux',
+		value: 22000,
+		valueDisplay: '22K',
+		percentage: 1,
+	},
+	{
+		label: 'Windows',
+		value: 80000,
+		valueDisplay: '80K',
+		percentage: 2,
+	},
+];
+
+/**
+ * Registry of all available sample datasets with metadata
+ */
+export const sampleDataRegistry: Record< string, SampleDataConfig > = {
```

Comment:
The metadata is useful, but I wonder if we could just have jsdoc describing each dataset. Being co-located would make it less likely to be missed or become outdated.

Reply:
Sounds good. I reckon keeping the JSDoc right on the export makes everything cleaner, easier to maintain, and way better for dev UX. Refactoring now.